### DISCUSSION TITLE
✅ 🔗 🚀 Feature Request: Lottie animation 'Hover Area' needs updating to include 'Container' as well as 'Column' and 'Section' (Duplicate of #20360) - [ED-9468]

### DISCUSSION THREAD

**Discussion by billieargent** [elementor/elementor#22207] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I have lotties on my website which animate when the column they are placed in is hovered over. Since updating the website to containers, I cannot create the same effect as the lottie is now in a container, not a column or section
![Screenshot 2023-04-26 150217](https://user-images.githubusercontent.com/119862146/234602994-18b78bfe-2b24-46c5-8f2b-0a8f811a9c57.png)

### Describe the solution you'd like

Please add 'Container' to the 'Hover Area' select options in the lottie widget.

### Describe alternatives you've considered

I can keep using sections/columns for the areas with lotties but I would like to convert these to containers.

**Comment by nicholaszein** [comment]
Hello there, @billieargent!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #20360.

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.15.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
